DATA/production/configurations/asyncReco/setenv_extra.sh

```diff
@@ -206,13 +206,23 @@ if [[ -z $RUN_IR ]] || [[ -z $RUN_DURATION ]] || [[ -z $RUN_BFIELD ]]; then
   export RUN_IR=`cat IR.txt`
   export RUN_DURATION=`cat Duration.txt`
   export RUN_BFIELD=`cat BField.txt`
+  export RUN_DETECTOR_LIST=`cat DetList.txt`
 fi
+echo "DETECTOR LIST for current run ($RUNNUMBER) = $RUN_DETECTOR_LIST"
+echo "DURATION for current run ($RUNNUMBER) = $RUN_DURATION"
+echo "B FIELD for current run ($RUNNUMBER) = $RUN_BFIELD"
 echo "IR for current run ($RUNNUMBER) = $RUN_IR"
 if (( $(echo "$RUN_IR <= 0" | bc -l) )); then
   echo "Changing run IR to 1 Hz, because $RUN_IR makes no sense"
   RUN_IR=1
 fi
-echo "Duration of current run ($RUNNUMBER) = $RUN_DURATION"
+
+# Let's check if ZDC is in the detector list; this is needed for TPC dist correction scaling in PbPb 2023
+SCALE_WITH_ZDC=1
+isZDCinDataTaking=`echo $RUN_DETECTOR_LIST | grep ZDC`
+isFT0inDataTaking=`echo $RUN_DETECTOR_LIST | grep FT0`
+[[ -z $isZDCinDataTaking ]] && SCALE_WITH_ZDC=0
+[[ -z $isFT0inDataTaking ]] && SCALE_WITH_FT0=0
```

Comment:
looks fine, but should not `SCALE_WITH_FT0` be declared?

Reply:
Yes, I forgot. Thanks!